bus/pubsub/consumer.go

```diff
@@ -28,6 +28,7 @@ type Consumer struct {
 
 func (o *Option) NewConsumer() (c *Consumer, err error) {
 	opts := make([]option.ClientOption, 0)
+	opts = append(opts, option.WithGRPCConnectionPool(1))
```

Comment:
Just remove the option all together, it will just use one connection if the option isn't provided